cloudscale/resource_cloudscale_floating_ip.go

```diff
@@ -117,7 +120,9 @@ func resourceFloatingIPRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("network", floatingIP.Network)
 	d.Set("next_hop", floatingIP.NextHop)
 	d.Set("reverse_ptr", floatingIP.ReversePointer)
-	d.Set("server", floatingIP.Server.UUID)
+	if floatingIP.Server != nil {
```

Comment:
```suggestion
	if floatingIP.Server.UUID != "" {
```

Reply:
This is now fixed by updating to the latest cloudscale-go-sdk.